vdb/lib/osv.py

```diff
@@ -55,7 +54,7 @@ def download_all(self):
         # For performance do not retain the whole data in-memory
         # See: https://github.com/AppThreat/vulnerability-db/issues/27
         data_list = []
-        for _, url in config.osv_url_dict.items():
+        for _, url in config.OSV_URL_DICT.items():
```

Comment:
This rename got missed out in a previous PR.

Reply:
@prabhu Good catch!